staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go

```diff
@@ -71,15 +74,8 @@ var (
 		},
 		[]string{"resource"},
 	)
-	dbTotalSize = compbasemetrics.NewGaugeVec(
-		&compbasemetrics.GaugeOpts{
-			Subsystem:      "apiserver",
-			Name:           "storage_db_total_size_in_bytes",
-			Help:           "Total size of the storage database file physically allocated in bytes.",
-			StabilityLevel: compbasemetrics.ALPHA,
-		},
-		[]string{"endpoint"},
-	)
```

Comment:
Removing this metric will be breaking change for some users, it would be safer if we leave a 1 release deprecation period before removing it completely.

Reply:
https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/3498-extending-stability#alpha-metrics

Reply:
Although this is what was decided when api stability was designed, we've been careful when removing old metrics without giving a grace period to the users.

IMO, we will need to continue being careful when removing alpha metrics and apply deprecation periods until we start moving metrics to beta.

I leave it up to @logicalhan to make a decision here.

Reply:
True, but I would argue that old metrics are treated more like Beta and not Alpha. SIG instrumentation should make it explicit and just move them to Beta to avoid making exceptions. For now I treat this as an Alpha metric, thus removing.

Reply:
I would deprecate it, people likely depend on this one.

Reply:
Will do